joint_calling/upload_processor.py

```diff
@@ -0,0 +1,81 @@
+""" This function prepares gCVF's uploaded to GCS, based off sample status
+    logged in a database, for further QC and downstream analysis.
+    The upload processor will determine when samples should be added
+    to existing MatrixTables where appropriate and which MatrixTables
+    they should be combined with in this case. Following a successful
+    run all uploaded files will be moved to archival storage."""
+
+from google.cloud import storage
+from google.api_core.exceptions import NotFound
+
+
+def copy_files(
+    sample_files: list, source_bucket_name: str, target_bucket_name: str
```

Comment:
- Can you use type hints to define what types are allowed in the list?
- Should `sample_files` contain the source bucket path?
- `destination` is a bit more common as the opposite of `source` than `target`, I think.

Reply:
I've updated the type hints now. I imported List from typing since we're using 3.7, hope this was the right thing to do?
I've switched target to the destination, thanks! 

I have just realized I haven't addressed your middle point yet! I think I need to reflect on this a little bit.

Reply:
Source bucket path is now taken in as a parameter! :D